spyder/widgets/panels/edgeline.py

```diff
@@ -61,19 +61,13 @@ def update_color(self):
         """
         self.color = self.editor.highlighter.get_color_name('comment')
 
-    def set_geometry(self, cr):
-        """Calculate and set geometry of edge line panel.
+    def geometry(self):
+        """Calculate the geometry of edge line panel.
             start --> fist line position
             width --> max position - min position
         """
-        width = self.editor.fontMetrics().width('9'*(max(self.columns) - min(self.columns))) + 1
-        offset = self.editor.contentOffset()
-        x = self.editor.blockBoundingGeometry(self.editor.firstVisibleBlock()) \
-            .translated(offset.x(), offset.y()).left() \
-            +self.editor.fontMetrics().width('9'*min(self.columns))+5
+        x = self.editor.fontMetrics().width('9'*min(self.columns))
+        width = self.editor.fontMetrics().width('9'*(max(self.columns)
+                                                - min(self.columns))) + 1
```

Comment:
So what is the 1 for :-p

Reply:
It's the width of an edge line

Reply:
If this is the case, then this might fail on other Operating systems. We should use Qt metrics then. otherwise you just changed a 5 for a 1

Reply:
That one was always there, I'll search the best way to make it platform independent

Reply:
If we don't set the QPen width it will be 1 pixel width. http://doc.qt.io/qt-4.8/qpen.html#details

What I should do? define a constant `WIDTH = 1`? add `width` as a property of the class?
 add a comment?

Reply:
Maybe?